<html><head><title>Programmer/Bioinformatician - Rochester, NY</title></head>
<body><h2>Programmer/Bioinformatician - Rochester, NY</h2>
<div><div><div><div><div><p>This is an exciting opportunity for a highly motivated bioinformatician to join a research team at the cutting edge of translational research with a particular focus on the viral and bacterial microbiome in health and infectious disease. The successful candidate would be required to use applied informatics to support wet lab scientists utilizing high-throughput sequencing and other data types for their research. The Monaco Lab routinely performs analysis of sequencing data including, but not limited to, whole genome sequencing, virome sequencing, 16s rRNA sequencing, and mRNA-seq. In addition, this position requires experience in QIIME2 and involves analytical pipeline development. Microbiome data includes information about viral and bacterial taxa between different individuals and how these taxa change over time and relate to disease, normal body functions, or experimental treatments. Genomic data from prokaryotic cells includes information about genomic variation between strains and global transcriptional responses to in vitro experimental conditions and in vivo infections or host response. Metagenomic data from bacterial communities includes 16S rRNA, metagenomic and metatranscriptomic profiles. Ability to extract and graphically represent complex data is a must. Excellent communication skills as well as accurate record keeping are essential. Includes bioinformatic data analyses and writing related to these analyses for publication in peer-reviewed journals.</p><p><b>
Responsibilities:</b></p><p>
Develop and optimize bioinformatics pipelines and other tools utilizing high-throughput sequencing data to identify sequences of interest, including viral sequences.</p><p>
Analysis of high-throughput sequencing data generated on next-generation sequencing instruments, including Illumina. The sequence data will include, but is not limited to: 1) metagenomic sequence data from experimental approaches such as whole genome sequencing, virus-specific sequencing, and 16s rRNA sequencing and 2) RNA-seq data from tissues, cells and bacteria. Analysis will include assessment of primary sequence data quality and evaluation of biological questions during downstream processing. Required tasks will include, but is not limited to, inputting samples into new and established pipelines (i.e. QIIME2) to determine microbial taxa; develop tools required to format large data sets, summarize the data and produce high quality figures for presentations and publications; use of version control software, such as github, and ensuring data reproducibility via RMarkdown documentation or similar. This requires an extensive working knowledge of linux/unix commands, formatting for SLURM on a shared server platform, computer programming languages including Perl and Python and application of open-source software tools, including QIIME2.</p><p>
Proactively seek out and evaluate new software for analysis of microbiome/metagenomic data. This may include active collaborations with bioinformaticians and programmers at URMC and other institutions. Development and publication of new software/pipelines for data analysis is encouraged. May involve developing tools to track and monitor sequence data quality and output for internal quality control and grant applications.</p><p>
Train lab members on how to use various software for analysis of sequencing data.</p><p>
Attend lab meetings, meeting with supervisors, attending educational seminars.</p><p><b>
Qualifications:</b></p><p>
Bachelor's degree in related discipline such as Computer Science, Business, Mathematics, Statistics, Science or Engineering, and 1-2 years of related experience; or an equivalent combination of education and experience.</p><p>
Preferred Qualifications</p><p>
A background in bioinformatics, computational biology, or a related field. Competency with multiple computer operating systems (Windows, Mac, Linux and Unix), computer languages (C++, Java/Parallel Java, MySQL, Perl, Python and R) is essential. Excellent interpersonal, verbal, and written communication skills; strong organizational and analytical skills; and the ability to work in a highly cooperative team environment as well as independently is required. Working knowledgebase of a variety of publically available tools required for various stages of analysis. Strong knowledge of basic statistics, linear model building, principal component analysis, and factor analysis required. Master's degree in Bioinformatics; documented prior research in molecular biology/virology, the microbiome or metabolome; or analysis, management, and visualization of high-throughput sequence data preferred.</p><br/>
</div>
</div></div>
</div><br/>
<div></div><div><div><div>How To Apply
</div></div>
</div><br/>
<div></div><div><div><div><div><br/>
<p>All applicants must apply online.</p><p></p><br/>
<p><i>
EOE Minorities/Females/Protected Veterans/Disabled</i></p></div></div></div></div></div></body>
</html>